src/store/ChartStore.js

```diff
@@ -269,7 +269,7 @@ class ChartStore {
         this.resizeObserver = new ResizeObserver(() => this.resizeScreen());
         this.resizeObserver.observe(rootNode);
 
-        stxx.append('createDataSet', this.updateComparisons);
+        this.feed.onComparisonDataUpdate(this.updateComparisons.bind(this));
```

Comment:
@brucebinary can we remove `.bind(this)` since updateComparisons already has `@action.bound` ?

Reply:
Ah. Got it.